### PR TITLE
new (rather old, but not merged yet) LennardJonesGenericCapped Potential 

### PR DESCRIPTION
**Pull request by stuehn**
some people used this potential already in the past, it just was not yet included in the master branch

### CODE REVIEW

File: src/Version.hpp [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 #define MAJORVERSION 1
 #define MINORVERSION 9
-#define PATCHLEVEL   3
+#define PATCHLEVEL   7
```
Comment (junghans) [inline src/Version.hpp]: Why are we bumping +4 and not +1?

File: src/interaction/LennardJonesGenericCapped.cpp [hunk 2]
```diff
@@ -0,0 +1,105 @@
+/*
+  Copyright (C) 2012,2013
```
Comment (junghans) [inline src/interaction/LennardJonesGenericCapped.cpp]: Year 2016 missing!

File: src/interaction/LennardJonesGenericCapped.hpp [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+          real energy = 4.0 * epsilon * (ffA - ffB);
+          return energy;
+        }
+      }
+
+      bool _computeForceRaw(Real3D& force, const Real3D& dist, real distSqr) const {
+
+        real capradSqr = caprad * caprad;
+        if (distSqr > capradSqr) {
+    	  real invdist = 1.0 / sqrt(distSqr);
```
Comment (junghans) [inline src/interaction/LennardJonesGenericCapped.hpp]: Wouldn't 
```
real invdist = 1.0/((distSqr > capradSqr)?sqrt(distSqr):caprad;
```
do the same?

**Comment by junghans** [comment]
Copyright update in all touched files missing.

File: src/interaction/LennardJonesGenericCapped.hpp [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+        caprad = _caprad;
+        updateAutoShift();
+        preset();
+      }
+
+      real getCaprad() const { return caprad; }
+
+
+        void setSigma(real _sigma) {
+            sigma = _sigma;
```
Comment (MrTheodor) [inline src/interaction/LennardJonesGenericCapped.hpp]: indentation

File: src/interaction/LennardJonesGenericCapped.py [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+                      interaction_VerletListHadressLennardJonesGenericCapped, \
+                      interaction_CellListLennardJonesGenericCapped, \
+                      interaction_FixedPairListLennardJonesGenericCapped
+
+class LennardJonesGenericCappedLocal(PotentialLocal, interaction_LennardJonesGenericCapped):
+    'The (local) generic Lennard-Jones potential with force capping.'
+    def __init__(self, epsilon=1.0, sigma=1.0, a=12, b=6,
+                 cutoff=infinity, caprad=0.0, shift="auto"):
+        """Initialize the local generic Lennard Jones object."""
+        if not (pmi._PMIComm and pmi._PMIComm.isActive()) or pmi._MPIcomm.rank in pmi._PMIComm.getMPIcpugroup():
```
Comment (MrTheodor) [inline src/interaction/LennardJonesGenericCapped.py]: I know, it's everywhere but pmi.workerIsActive() does the same and it's shorter

**Comment by junghans** [comment]
And a test would be nice....

**Comment by govarguz** [comment]
ping @stuehn

**Comment by junghans** [comment]
@stuehn did you not wanted to merge this potential?

**Pull request closed without merging**